component-versions.md

```diff
@@ -2,6 +2,76 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated this is called out in the [release notes](https://discourse.ubuntu.com/t/release-notes/17842) and components are marked with `n/a` below.
 
+[Details=1.13.0]
+
+### Charms
+
+| Name | Revision |
+|----------|--------------|
+| cs:~anbox-charmers/lxd   |  99 |
```

Comment:
Should we stick to the old charm names for now and update them here together with the rest of the docs? Or should we start using the new ones here now since you'll now get the new ones?

Reply:
As discussed we stick to the old names for 1.13 and migrate to the new ones with 1.14